3_extracting_features/DPutils.py

```diff
@@ -0,0 +1,220 @@
+from importlib.resources import path
+import pandas as pd
+import pathlib
+import shutil
+
+from PIL import Image
+import os
+
+def copy_DP_files(
+    project_path: pathlib.Path,
+    config_name: str,
+    checkpoint_name: str,
+):
+    """Copy config and checkpoint files to their designated location in DP project as assigned by the project_path
```

Comment:
Why is it important to copy files? Why not just use them where they are, or have them in the right place to start?

Can you answer this question in the function docstring?

Reply:
The answer is now within the doc string. I think it is easier to just have code that put everything in the right place then manually doing it, plus it shows where certain files need to be saved to.